test/alternator/test_metrics.py

```diff
@@ -191,6 +191,14 @@ def test_wcu(test_table_s, metrics):
         val2 = 'a' * (2 * KB - total_length + 1) # message length 2K + 1
         test_table_s.put_item(Item={'p': p, 'att': val, 'another': val2})
 
+def test_wcu_batch_write_items(test_table_s, metrics):
```

Comment:
Again. please use the accurate operation name, BatchWriteItem (singular), in the commit title, text and test name. It's not a big deal but it's one small thing to reduce confusion what is the correct name of this operation.